### PR TITLE
Update test registration screen to new design. (EXPOSUREAPP-7229)

### PR DESCRIPTION
**Pull request by SamuraiKek**

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml [hunk 1]
```diff
@@ -46,29 +46,7 @@
...
                     app:layout_constraintTop_toTopOf="parent"
-                    app:srcCompat="@drawable/ic_illustration_test" />
-
-                <TextView
-                    android:id="@+id/submission_dispatcher_text"
-                    style="@style/subtitle"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/spacing_normal"
-                    android:text="@string/submission_dispatcher_subheadline"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml:57]: could we delete the un used string?

File: Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml [hunk 2]
```diff
@@ -46,29 +46,7 @@
...
-                    app:layout_constraintStart_toStartOf="@+id/guideline_start"
-                    app:layout_constraintTop_toBottomOf="@+id/submission_dispatcher_illustration" />
-
-                <TextView
-                    android:id="@+id/submission_dispatcher_needs_testing_text"
-                    style="@style/headline6"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/spacing_normal"
-                    android:text="@string/submission_dispatcher_needs_testing_subheadline"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml:68]: Also here?

**Comment by mtwalli** [comment]
Tested on  Samsung S9 and looks good 👍

**Review by mtwalli — APPROVED** [review]

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Pull request merged**